Sources/StreamChatSwiftUI/ChatChannel/Composer/Suggestions/CommandsConfig.swift

```diff
@@ -33,23 +33,41 @@ public class DefaultCommandsConfig: CommandsConfig {
     public func makeCommandsHandler(
         with channelController: ChatChannelController
     ) -> CommandsHandler {
-        let mentionsCommand = MentionsCommandHandler(
+        let mentionsCommandHandler = MentionsCommandHandler(
             channelController: channelController,
             commandSymbol: mentionsSymbol,
             mentionAllAppUsers: false
         )
-        let giphyCommand = GiphyCommandHandler(commandSymbol: "/giphy")
-        let muteCommand = MuteCommandHandler(
-            channelController: channelController,
-            commandSymbol: "/mute"
-        )
-        let unmuteCommand = UnmuteCommandHandler(
-            channelController: channelController,
-            commandSymbol: "/unmute"
-        )
-        let instantCommands = InstantCommandsHandler(
-            commands: [giphyCommand, muteCommand, unmuteCommand]
+        
+        var instantCommands = [CommandHandler]()
+        
+        let channelConfig = channelController.channel?.config
+        
+        let giphyEnabled = channelConfig?.commands.first(where: { command in
+            command.name == "giphy"
+        }) != nil
+        
+        if giphyEnabled {
+            let giphyCommand = GiphyCommandHandler(commandSymbol: "/giphy")
+            instantCommands.append(giphyCommand)
+        }
+        
+        if channelConfig?.mutesEnabled == true {
```

Comment:
Is the `== true` added for readability? Could be dropped and I think would still be okay.

Reply:
....nevermind, it is required. :)